src/NuGet.Core/NuGet.Common/Telemetry/TelemetryActivity.cs

```diff
@@ -9,7 +9,7 @@ namespace NuGet.Common
 {
     /// <summary> Represents telemetry activity which spans a time interval. </summary>
     /// <remarks> Always dispose the activity at the end of interval covered. </remarks>
-    public class TelemetryActivity : IDisposable
+    public sealed class TelemetryActivity : IDisposable
```

Comment:
I thought these were not going to be sealed.

Reply:
I left this one as is assuming that this class may not be inherited other folks. I will add a protected member instead.